pypeit/reduce.py

```diff
@@ -412,13 +424,17 @@ def run(self, basename=None, ra=None, dec=None, obstime=None,
         # Finish up
         if self.sobjs.nobj == 0:
             msgs.warn('No objects to extract!')
-        else:
+        elif self.par['flexure']['spec_method'] not in ['skip', 'slit_cen']:
```

Comment:
I can imagine that someone would want to do both slit method and the object methods, so you should make that a possibility. In particular running the object flexure code after your slit flexure runs would be a good way of checking that whether the object corrections are even necessary.

Reply:
Okay good.